app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
```

Comment:
This could lead to confusion if the user modifies the date range then clicks the link without first submitting the search form?